src/app/dashboard/help/ai-board/page.jsx

```
@@ -0,0 +1,168 @@
+                    <AvatarFallback>CN</AvatarFallback>
+                  </Avatar>
+                )}
+                <div
+                  className={`p-3 rounded-md max-w-[80%] text-sm ${
+                    message.role === "user"
+                      ? "bg-teal-100 text-teal-800"
+                      : "bg-gray-200 text-gray-800"
+                  }`}
+                >
+                  <ReactMarkdown
+                    children={message.content}
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)